x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/models_list.tsx

```diff
@@ -0,0 +1,469 @@
+        title: i18n.translate('xpack.ml.inference.modelsList.fetchDeletionErrorMessage', {
+          defaultMessage: '{modelsCount, plural, one {Model} other {Models}} deletion failed',
+          values: {
+            modelsCount: modelsToDeleteIds.length,
+          },
+        }),
+      });
+    }
+  }
+
+  const actions: Array<Action<ModelItem>> = [
+    {
+      name: i18n.translate('xpack.ml.inference.modelsList.viewTrainingDataActionLabel', {
+        defaultMessage: 'View training data',
+      }),
+      description: i18n.translate('xpack.ml.inference.modelsList.viewTrainingDataActionLabel', {
+        defaultMessage: 'View training data',
+      }),
+      icon: 'list',
+      type: 'icon',
+      available: (item) => item.metadata?.analytics_config?.id,
+      onClick: async (item) => {
+        await navigateToUrl(
+          getResultsUrl(
+            item.metadata?.analytics_config.id,
+            Object.keys(item.metadata?.analytics_config.analysis)[0]
+          )
+        );
+      },
+      isPrimary: true,
+    },
+    {
+      name: i18n.translate('xpack.ml.inference.modelsList.deleteModelActionLabel', {
+        defaultMessage: 'Delete model',
+      }),
+      description: i18n.translate('xpack.ml.inference.modelsList.deleteModelActionLabel', {
+        defaultMessage: 'Delete model',
+      }),
+      icon: 'trash',
+      type: 'icon',
+      color: 'danger',
+      isPrimary: false,
+      onClick: async (model) => {
+        await prepareModelsForDeletion([model]);
+      },
+      enabled: (item) => {
+        return !item.pipelines;
+      },
+    },
+  ];
+
+  const toggleDetails = async (item: ModelItem) => {
+    const itemIdToExpandedRowMapValues = { ...itemIdToExpandedRowMap };
+    if (itemIdToExpandedRowMapValues[item.model_id]) {
+      delete itemIdToExpandedRowMapValues[item.model_id];
+    } else {
+      await fetchModelsStats([item]);
+      itemIdToExpandedRowMapValues[item.model_id] = <ExpandedRow item={item as ModelItemFull} />;
+    }
+    setItemIdToExpandedRowMap(itemIdToExpandedRowMapValues);
+  };
+
+  const columns: Array<EuiBasicTableColumn<ModelItem>> = [
+    {
+      align: 'left',
+      width: '40px',
+      isExpander: true,
+      render: (item: ModelItem) => (
+        <EuiButtonIcon
+          onClick={toggleDetails.bind(null, item)}
+          aria-label={
+            itemIdToExpandedRowMap[item.model_id]
+              ? i18n.translate('xpack.ml.inference.modelsList.collapseRow', {
+                  defaultMessage: 'Collapse',
+                })
+              : i18n.translate('xpack.ml.inference.modelsList.expandRow', {
+                  defaultMessage: 'Expand',
+                })
+          }
+          iconType={itemIdToExpandedRowMap[item.model_id] ? 'arrowUp' : 'arrowDown'}
+        />
+      ),
+    },
+    {
+      field: ModelsTableToConfigMapping.id,
+      name: i18n.translate('xpack.ml.inference.modelsList.modelIdHeader', {
+        defaultMessage: 'Model ID',
```

Comment:
Can this just be `ID` as the `Model` part is implied by being on the Models tab. We don't use `Job ID` for example on the Jobs tab.

Reply:
Fixed in https://github.com/elastic/kibana/pull/74978/commits/54ced0c1468f3b0dbb4847e11643013791375bb6